ci-operator/step-registry/hypershift/kubevirt/install/disconnected/hypershift-kubevirt-install-disconnected-commands.sh

```diff
@@ -0,0 +1,267 @@
+oc_mirror_image="quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:278b6167e214992b2a40dd2fb44e8588f4a9ef100a70ec20cada58728350dd02"
+oc image extract \$oc_mirror_image --path /usr/bin/oc-mirror:/home --confirm
+if ls /home/oc-mirror >/dev/null ;then
+    chmod +x /home/oc-mirror
+else
+    echo "Warning, can not find oc-mirror abort !!!"
+    exit 1
+fi
+
+echo "6. oc-mirror --config /home/imageset-config.yaml docker://\${mirror_registry} --oci-registries-config=/home/registry.conf --continue-on-error --skip-missing"
+catalog_image="openshift-cnv/nightly-catalog" # TODO: handle stable when needed
+
+cat <<END |tee "/home/registry.conf"
+[[registry]]
+ location = "registry.redhat.io/rhacm2"
+ insecure = true
+ blocked = false
+ mirror-by-digest-only = false
+ [[registry.mirror]]
+    location = "quay.io:443/acm-d"
+    insecure = true
+[[registry]]
+ location = "registry.redhat.io/multicluster-engine"
+ insecure = true
+ blocked = false
+ mirror-by-digest-only = false
+ [[registry.mirror]]
+    location = "quay.io:443/acm-d"
+    insecure = true
+[[registry]]
+ location = "registry.access.redhat.com/openshift4/ose-oauth-proxy"
+ insecure = true
+ blocked = false
+ mirror-by-digest-only = false
+ [[registry.mirror]]
+    location = "registry.redhat.io/openshift4/ose-oauth-proxy"
+    insecure = true
+[[registry]]
+ location = "registry.stage.redhat.io"
+ insecure = true
+ blocked = false
+ mirror-by-digest-only = false
+ [[registry.mirror]]
+    location = "brew.registry.redhat.io"
+    insecure = true
+[[registry]]
+ location = "registry-proxy.engineering.redhat.com/rh-osbs"
+ insecure = true
+ blocked = false
+ mirror-by-digest-only = false
+ [[registry.mirror]]
+    location = "brew.registry.redhat.io/rh-osbs"
+    insecure = true
+END
+
+cat <<END |tee "/home/imageset-config.yaml"
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v1alpha2
+storageConfig:
+  local:
+    path: mirror
+mirror:
+  operators:
+  - catalog: "oci:///home/cnv-local-catalog"
+    targetCatalog: \${catalog_image}
+    targetTag: "\${CNV_PRERELEASE_VERSION}"
+    packages:
+    - name: kubevirt-hyperconverged
+      channels:
+      - name: \${CNV_CHANNEL}
+END
+
+pushd /home
+echo "Because of OCPBUGS-20137, it's necessary to run multiple times."
+/home/oc-mirror --config "/home/imageset-config.yaml" docker://\${mirror_registry}  --include-local-oci-catalogs --oci-registries-config="/home/registry.conf" --continue-on-error --skip-missing
+/home/oc-mirror --config "/home/imageset-config.yaml" docker://\${mirror_registry}  --include-local-oci-catalogs --oci-registries-config="/home/registry.conf" --continue-on-error --skip-missing
+/home/oc-mirror --config "/home/imageset-config.yaml" docker://\${mirror_registry}  --include-local-oci-catalogs --oci-registries-config="/home/registry.conf" --continue-on-error --skip-missing
+RESULTS_FILE=\$(find oc-mirror-workspace -type f -name 'catalogSource-nightly-catalog.yaml')
+oc apply -f "\$RESULTS_FILE"
+popd
```

Comment:
oc-mirror:v4.13.9 is not stable, so we need change to v4.14.1, There are some differences in their installation methods
https://github.com/openshift/release/blob/34ffb1819242e23b2c733e902414dc1dc95523d5/ci-operator/step-registry/hypershift/mce/install/disconnected/hypershift-mce-install-disconnected-commands.sh#L124-L126